src/sdkV2.ts

```diff
@@ -111,8 +124,10 @@ export default class CegaEvmSDKV2 {
     asset: EvmAddress,
     overrides: TxOverrides = {},
   ): Promise<ethers.providers.TransactionResponse> {
+    const addressManager = this.loadAddressManager();
+    const cegaEntryAddress = await addressManager.getCegaEntry();
```

Comment:
nit:
```
const cegaEntry = await this.loadCegaEntry()
```
then use `cegaEntry.address`

Allows us to reduce access to the address manager when not needed

Reply:
ahhh nicee yeh that's better